packages/api-gateway/__tests__/geolocation/deletePlace.test.js

```diff
@@ -0,0 +1,143 @@
+const Fastify = require('fastify');
+const Autoload = require('fastify-autoload');
+const pluginLoader = require('fastify-plugin');
+const boom = require('@hapi/boom');
+const { faker } = require('@faker-js/faker');
+const getApiGatewayDirectory = require('../../src/utils/getApiGatewayDirectory');
+
+describe('[DELETE] geolocation/place endpoint then return id and status code 200', () => {
+  // mock deletePlace method
+  const deletePlace = () => async (id) => id;
+  const GeolocationServices = {
+    deletePlace: deletePlace(),
+  };
+
+  async function services(fastify) {
+    await fastify.decorate('geolocationServices', GeolocationServices);
+  }
+
+  // load fastify and update function
+  const fastify = Fastify({ logger: true });
+  fastify.register(Autoload, { dir: getApiGatewayDirectory() });
+  fastify.register(pluginLoader(services));
```

Comment:
@CETHER as we are doing this in all tests, why don't we move it externally? I mean what we can do is to create an external function that does this, or we can create another `describe` that covers the describes that we have here and then we can use the `beforeEach` statement to set up the server.

Reply:
Done. I mock a function externally to build the fastify app. I will learn how to use beforeEach of jest and I will try to apply this later.